src/test/java/com/vinylteam/vinyl/dao/jdbc/JdbcVinylDaoITest.java

```diff
@@ -0,0 +1,410 @@
+        assertEquals("best artist", allVinyls.get(1).getArtist());
+        assertEquals("cool release (Vinyl) - best artist", allVinyls.get(1).getFullNameVinyl());
+        assertEquals("pop", allVinyls.get(1).getGenre());
+        assertEquals("200 uah", allVinyls.get(1).getPrice());
+        assertEquals("https://vinylsite.com/there/coolreleasevinyl", allVinyls.get(1).getVinylLink());
+        assertEquals("https://imagestore.com/there/imagerelease.jpg", allVinyls.get(1).getImageLink());
+        assertEquals(2, allVinyls.get(1).getShopId());
+        assertEquals(1, allVinyls.get(1).getUniqueVinylId());
+
+        assertEquals(3, allVinyls.get(2).getVinylId());
+        assertEquals("wow effect", allVinyls.get(2).getRelease());
+        assertEquals("best of the best", allVinyls.get(2).getArtist());
+        assertEquals("wow effect - best of the best", allVinyls.get(2).getFullNameVinyl());
+        assertEquals("rock", allVinyls.get(2).getGenre());
+        assertEquals("1500 euro", allVinyls.get(2).getPrice());
+        assertEquals("https://bestvinyl.com/here/woweffect", allVinyls.get(2).getVinylLink());
+        assertEquals("https://imagestore.com/somewhere/wow.jpg", allVinyls.get(2).getImageLink());
+        assertEquals(1, allVinyls.get(2).getShopId());
+        assertEquals(2, allVinyls.get(2).getUniqueVinylId());
+    }
+
+    @Test
+    void getUniqueVinylByIdITest() {
+        Vinyl uniqueVinyl = vinylDao.getUniqueVinylById(2);
+        assertEquals(2, uniqueVinyl.getVinylId());
+        assertEquals("wow effect", uniqueVinyl.getRelease());
+        assertEquals("best of the best", uniqueVinyl.getArtist());
+        assertEquals("wow effect - best of the best", uniqueVinyl.getFullNameVinyl());
+        assertEquals("https://imagestore.com/somewhere/wow.jpg", uniqueVinyl.getImageLink());
+    }
+
+    @Test
+    void getUniqueVinylByNonExistedIdITest() {
+        assertThrows(RuntimeException.class, () -> {
+            vinylDao.getUniqueVinylById(15);
+        });
+    }
+
+    @Test
+    void getVinylByIdITest() {
+        Vinyl vinyl = vinylDao.getVinylById(2);
+        assertEquals(2, vinyl.getVinylId());
+        assertEquals("cool release (Vinyl)", vinyl.getRelease());
+        assertEquals("best artist", vinyl.getArtist());
+        assertEquals("cool release (Vinyl) - best artist", vinyl.getFullNameVinyl());
+        assertEquals("pop", vinyl.getGenre());
+        assertEquals("200 uah", vinyl.getPrice());
+        assertEquals("https://vinylsite.com/there/coolreleasevinyl", vinyl.getVinylLink());
+        assertEquals("https://imagestore.com/there/imagerelease.jpg", vinyl.getImageLink());
+        assertEquals(2, vinyl.getShopId());
+        assertEquals(1, vinyl.getUniqueVinylId());
+    }
+
+    @Test
+    void getVinylByNonExistedIdITest() {
+        assertThrows(RuntimeException.class, () -> {
+            vinylDao.getVinylById(15);
+        });
+    }
+
```

Comment:
Add getVinylsFromEmptyTableTest and getUniqueVinylsFromEmptyTableTest - and use truncate statement there.

Reply:
what result do we expect? RuntimeException?